dbscripts/xml/upgrade/event_log_oneclickuserid.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * event_log_oneclickuserid.xml
+  *
+  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2003-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Add missing user id on event logs from 'one-click' reviews.
+  *
+  -->
+
+<data>
+	<sql>
+		<query>
+			UPDATE event_log el JOIN ( event_log_settings els JOIN review_assignments ra ) ON ( els.log_id=el.log_id AND els.setting_name='reviewId' AND ra.review_id=els.setting_value )
```

Comment:
I think this query may have to be written separately for PostgreSQL and MySQL. Suggestions:
- Break it into separate implementations for the JOIN syntax, e.g.:
  ```
  <query driver="mysql">...</query>
  <query driver="mysqli">...</query>
  <query driver="postgres7">...</query>
  ```
  (The `mysqli` and `mysql` implementations should be the same, just for different driver names.)
- Use decimal numbers instead of hex, but include the constant names for the purposes of grepping the code, e.g.:
  ```
  <query ...>... WHERE el.event_type IN (1073741830, ...)</query><!-- SUBMISSION_LOG_REVIEW_ACCEPT -->
  ```